### PR TITLE
feat(doc-2064): make examples functional in api client

### PR DESCRIPTION
**Pull request by amritk**
Made requests functional. We still have the double enter problem but I will leave that for another PR.
- fixed the phantom galaxy workspace
- built out the request example command palette
- added add and delete for examples and requests
- added routing for examples
- switched back to routerLink instead of @click handler on sidebar

### CODE REVIEW

File: packages/api-client/src/components/CommandPalette/CommandPaletteExample.vue [hunk 1]
```diff
@@ -7,24 +7,54 @@ import {
...
+  activeWorkspaceRequests,
+  requests,
+  requestExampleMutators,
+} = useWorkspace()
+
 const exampleName = ref('')
-const selectedRequest = ref(activeWorkspaceRequests.value[0])
+const selectedRequest = ref(
+  // Ensure we pre-select the correct request
+  requests[props.metaData ?? ''] ?? activeRequest.value,
```
Comment (marclave) [inline packages/api-client/src/components/CommandPalette/CommandPaletteExample.vue:35]: `requests[props.metaData ?? '']` wont this error? ohhh but then it defaults

**Review by marclave — APPROVED** [review]
literally woz energy, tested across all our examples + apps 🫡 

he just can't be stopped

File: packages/api-client/src/components/CommandPalette/CommandPaletteExample.vue (hunk 1 above)
Comment (amritk) [reply packages/api-client/src/components/CommandPalette/CommandPaletteExample.vue:35]: Up this bit allows us to delete all requests. We can probably add an empty state later

Comment (amritk) [reply packages/api-client/src/components/CommandPalette/CommandPaletteExample.vue:35]: Yep exactly. It will default

**Pull request merged**